aries-site/src/examples/templates/forms/RequiredFieldsExample.js

```diff
@@ -43,15 +43,21 @@ const FormContainer = ({ ...rest }) => {
 
 const Error = ({ children, ...rest }) => {
   return (
-    <Box align="center" background="background-front" gap="xsmall" direction="row"  {...rest} >
+    <Box
+      background="background-front"
+      align="center"
```

Comment:
Should this be align start to match designs? Here is image of design:
<img width="571" alt="Screen Shot 2020-08-04 at 12 00 07 PM" src="https://user-images.githubusercontent.com/12522275/89333675-23840500-d64a-11ea-8fd4-2d1463f3a2b1.png">

Reply:
@halocline  and @britt6612 to speak about in the AM. I'd like to know what the resolution was here.